recipes/sktime/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "sktime" %}
+{% set version = "0.4.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f066ce462b5f6013391ec7697471a22e6f67af70b85e670202fed02ac87488b1
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    
+  host:
+    - pip
+    - python <=3.7
+    - numpy >=1.18.0
+    - libgomp
+    - pandas >=1.0.0
+    - scikit-learn >=0.23.0
+    - statsmodels >=0.11.0
+    - cython
+    - wheel
+    
+  run:
+    - pip
```

Comment:
Is `pip`, `wheel` and `cython` required to run the package. If not, please remove them